frontend/public/module/k8s/index.ts

```diff
@@ -381,6 +411,20 @@ export type MachineSetKind = {
   };
 } & K8sResourceKind;
 
+export type Patch = {
```

Comment:
Consider updating kubevirt to use this so we don't declare it twice. (Or we could open an issue and ask the kubevirt team who can test.)

https://github.com/spadgett/console/blob/4b9b5037eb88c3de99d692c308c008d08e76144b/frontend/packages/kubevirt-plugin/src/types/types.ts#L9-L14

Reply:
Done